components/applications-service/pkg/storage/postgres/schema/sql/03_service_group_view.up.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE service ADD COLUMN full_pkg_ident TEXT NOT NULL DEFAULT '';
```

Comment:
What happens with existing records? 

Example: How would this impact our dev environment?  https://a2-perf-test-single-local-inplace-upgrade-dev.cd.chef.co/applications